src/main/java/com/volasoftware/tinder/controllers/CustomAsyncExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.volasoftware.tinder.controllers;
+
+import java.lang.reflect.Method;
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+
+public class CustomAsyncExceptionHandler implements AsyncUncaughtExceptionHandler {
+
+    @Override
+    public void handleUncaughtException(
+        Throwable throwable, Method method, Object... obj) {
+
+        System.out.println("Exception message - " + throwable.getMessage());
+        System.out.println("Method name - " + method.getName());
```

Comment:
If you want to log something, System.out.println() is not good option, because the output will be printed into the console (which is good only for local debugging). If you are not familiar with logging concept, search for any java Logger (ex link: https://betterstack.com/community/guides/logging/best-java-logging-libraries/)